cinema/serializers.py

```diff
@@ -1 +1,139 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = (
+            "id",
+            "first_name",
+            "last_name",
+            "full_name",
+        )
+
+
+class ActorDetailedSerialiser(serializers.ModelSerializer):
```

Comment:
Typo: The class name `ActorDetailedSerialiser` should be corrected to `ActorDetailedSerializer` to maintain consistency with the naming convention used elsewhere in the file.